modules/sdk/project-templates/project-templates-activator/src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import java.nio.file.Files
+import java.nio.file.Paths
+
+def projectDir = Paths.get(request.getOutputDirectory(), request.getArtifactId())
```

Comment:
If I'm not mistaken, in Groovy you can use properties without calling the getter. I mean `request.outputDirectory` instead of `request.getOutputDirectory()`.